.github/workflows/static-analysis.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - run: git branch --track main origin/main
         if: ${{ github.event_name == 'pull_request' }}
 
-      - run: npx nx affected -t lint test build
+      - run: npx nx affected -t lint test build --base=origin/main --head=${{ github.sha }}
```

Comment:
I think (🤞) these should be the built-in default base and head options for nx, was there an issue with the build that meant we had to specify them?

Reply:
Amended :)